src/client/components/Module/formatDate.js

```diff
@@ -0,0 +1,21 @@
+function formatDate(newDate) {
```

Comment:
the newDate parameter name is not the best, it is just a simple date what we want to format.